webextensions/api/storage.json

```
@@ -349,6 +352,207 @@
                 "version_added": "15"
               }
             }
+          },
+          "clear": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/clear",
+              "support": {
+                "chrome": {
+                  "version_added": "≤58"
+                },
+                "edge": {
+                  "version_added": "14"
+                },
+                "firefox": {
+                  "version_added": "45"
+                },
+                "firefox_android": {
+                  "version_added": "48"
+                },
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "14"
+                },
+                "safari_ios": {
+                  "version_added": "15"
+                }
+              }
+            }
+          },
+          "get": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/get",
+              "support": {
+                "chrome": {
+                  "version_added": "≤58"
+                },
+                "edge": {
+                  "version_added": "14"
+                },
+                "firefox": {
+                  "version_added": "45"
+                },
+                "firefox_android": {
+                  "version_added": "48"
+                },
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "14"
+                },
+                "safari_ios": {
+                  "version_added": "15"
+                }
+              }
+            },
+            "empty_key": {
+              "__compat": {
+                "description": "Supports empty key",
+                "support": {
+                  "chrome": {
+                    "version_added": "≤105"
+                  },
+                  "edge": {
+                    "version_added": "14"
+                  },
+                  "firefox": {
+                    "version_added": "45"
+                  },
+                  "firefox_android": {
+                    "version_added": "48"
+                  },
+                  "opera": "mirror",
+                  "safari": {
+                    "version_added": false
+                  },
+                  "safari_ios": "mirror"
+                }
+              }
+            }
+          },
+          "getBytesInUse": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/getBytesInUse",
+              "support": {
+                "chrome": {
+                  "version_added": "≤58"
+                },
+                "edge": {
+                  "version_added": "14"
+                },
+                "firefox": {
+                  "version_added": "78"
```

```suggestion
                  "version_added": false,
                  "impl_url": "https://bugzil.la/1385832"
```


this entry is storage.local.getBytesInUse, which should be marked as unsupported.